micropub.php

```diff
@@ -432,18 +452,21 @@ private static function postprocess($post_id) {
       $geo = explode(':', $geo);
       $geo = explode(';', $geo[0]);
       $coords = explode(',', $geo[0]);
-      update_post_meta($post_id, 'geo_latitude', trim($coords[0]));
-      update_post_meta($post_id, 'geo_longitude', trim($coords[1]));
+      $args['meta_input']['geo_latitude'] = trim($coords[0]);
```

Comment:
> The feature wasn't added till 4.4 though, which bumps the minimum version number to 4.4. But no one should be running a version older than 2 releases back anyway and be supported.

tough talk! :P

right now we only require >=3.0.1: https://wordpress.org/plugins/micropub/ . bumping that up to 4.4 is a big jump, but i'm ok with it if you think we should.